sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -188,4 +199,10 @@ Additionally, since a message cannot be settled if the receiver that was used to
   await serviceBusAdministrationClient.deleteRule();
   ```
 
+### Upcoming features
+
+- An optional method on `ServiceBusSender` to allow pre-initializing the sender link to remove the upfront cost from the first send operation.
```

Comment:
Let's re-order this and put it in the end

Reply:
Done